src/engine/SmtCore.cpp

```diff
@@ -389,6 +394,14 @@ PiecewiseLinearConstraint *SmtCore::chooseViolatedConstraintForFixing( List<Piec
     return candidate;
 }
 
+void SmtCore::pickBranchPLConstraint()
+{
+    if ( _needToSplit && ( !_constraintForSplitting ) )
```

Comment:
remove extra ( ).

Also, so far our terminology has been "splitting", not "branching". Does branching have a different meaning?

Reply:
renamed to "split"